client-src/elements/chromedash-gate-column.js

```diff
@@ -302,18 +314,37 @@ export class ChromedashGateColumn extends LitElement {
              this.user.editable_features.includes(this.feature.id)));
   }
 
+  renderAction(processStage, action) {
+    const label = action.name;
+    const url = action.url
+      .replace('{feature_id}', this.feature.id)
+      .replace('{outgoing_stage}', processStage.outgoing_stage);
+
+    return html`
+      <sl-button href=${url} target="_blank"
+       pill size=small variant=primary
+       >${label}</sl-button>
+    `;
+  }
+
   renderReviewStatusPreparing() {
-    if (this.userCanRequestReview()) {
-      return html`
-       <sl-button pill size=small variant=primary
-         @click=${this.handleReviewRequested}
-         >Request review</sl-button>
-      `;
-    } else {
+    if (!this.userCanRequestReview()) {
       return html`
         Review has not been requested yet.
       `;
     }
+
+    const processStage = findProcessStage(this.stage, this.process);
+    if (processStage?.actions?.length > 0) {
+      return processStage.actions.map(act =>
```

Comment:
Enclose this part with a <div> </div>? html` <div> ${processStage.actions.map(...)}</div>`

Reply:
Also specify the layout here when there are multiple processStage.action

Reply:
So far we only ever have zero or one action.  And, I think that regular flow layout works fine for these buttons.